client/functions.lua

```diff
@@ -10,6 +10,28 @@ function NDCore.Functions.GetCharacters()
     return NDCore.Characters
 end
 
+
+function NDCore.Functions.GetPlayersFromCoords(coords, distance)
+    local players = GetActivePlayers()
+    local ped = PlayerPedId()
```

Comment:
ped variable is not used unless coords aren't there so it's better if it's moved below line 19

Reply:
ped var moved to where initialize